### PR TITLE
improved developer page

### PR DESCRIPTION
**Pull request by Sharma-palak**
@bvishal8510  sir plzz review

### CODE REVIEW

File: core/views.py [hunk 1]
```diff
@@ -88,61 +90,112 @@ def get(self, request, *args, **kwargs):
         return render(request, self.template_name, context={'page': page, 'display_name': display_name,
                                                             'files':files, 'images':images})
 
-class BookAndAuthorsView(View):
+class BookAndAuthors_level1_View(View):
     template_name = 'common_page.html'
 
     def get(self, request, *args, **kwargs):
-        display_name = "<div class='col-lg-12 mx-auto '><h3 class=' my-2'><span class='about-us'><b>Book And Authors</b></span></h3></div>"
+        display_name = "<div class='col-lg-12 mx-auto '><h3 class=' my-2'><span class='about-us'><b>Book And Authors Second Year</b></span></h3></div>"
```
Comment (bvishal8510) [inline core/views.py]: change name to "Unit-wise PPT for Fourth Year", in this and every name below

File: core/views.py [hunk 2]
```diff
@@ -88,61 +90,112 @@ def get(self, request, *args, **kwargs):
...
         files = Files.objects.filter(page=page)
         images = Image.objects.filter(page=page)
         return render(request, self.template_name, context={'page': page, 'display_name': display_name,
                                                             'files':files, 'images':images})
+class BookAndAuthors_level2_View(View):
+    template_name = 'common_page.html'
 
-class UnitWiseNotesView(View):
+    def get(self, request, *args, **kwargs):
+        display_name = "<div class='col-lg-12 mx-auto '><h3 class=' my-2'><span class='about-us'><b>Book And Authors Fourth Year</b></span></h3></div>"
```
Comment (bvishal8510) [inline core/views.py]: change name to "Unit-wise PPT for Fourth Year" here, similarly all others

File: core/urls.py [hunk 3]
```diff
@@ -13,11 +15,16 @@
     url(r'^establishment/$', EstablishmentView.as_view(), name='establishment'),
     url(r'^structure/$', StructureView.as_view(), name='structure'),
     url(r'^syllabus/$', SyllabusView.as_view(), name='syllabus'),
-    url(r'^book-and-authors/$', BookAndAuthorsView.as_view(), name='book-and-authors'),
-    url(r'^unit-wise-notes/$', UnitWiseNotesView.as_view(), name='unit-wise-notes'),
-    url(r'^video-lectures/$', VideoLecturesView.as_view(), name='video-lectures'),
-    url(r'^unit-wise-ppt/$', UnitWisePPTView.as_view(), name='unit-wise-ppt'),
-    url(r'^UT-question-paper/$', UTQuestionPaperView.as_view(), name='UT-question-paper'),
+    url(r'^book-and-authors-level1/$', BookAndAuthors_level1_View.as_view(), name='book-and-authors-level1'),
+   url(r'^book-and-authors-level2/$', BookAndAuthors_level2_View.as_view(), name='book-and-authors-level2'),
```
Comment (bvishal8510) [inline core/urls.py]: all urls starting should be aligned

File: core/templates/common_page.html [hunk 4]
```diff
@@ -1,174 +1,176 @@
...
+                    <img src="{{media_link}}{{image.image.url}}" class="img-fluid " alt="image" >
+                </div>
+                {% endfor %}
+            </div>
+            {% endif %}
+            
+            
+            {% if page %}
+            <div class="row">
+                <div class="col-sm-6">
```
Comment (bvishal8510) [inline core/templates/common_page.html]: let this be col-12 not col-6

**Comment by Sharma-palak** [comment]
@bvishal8510  sir plzz review

File: core/templates/developer.html [hunk 5]
```diff
@@ -1,88 +1,117 @@
...
+            <li class="list-inline-item"><a href="https://www.facebook.com/methi40" target="_blank" class="social-link"><i class="fab fa-facebook-f"></i></a></li>
+            <li class="list-inline-item"><a href="https://www.github.com/methi40" target="_blank" class="social-link"><i class="fab fa-github"></i></a></li>
+            <li class="list-inline-item"><a href="https://www.instagram.com/mr_methi" target="_blank" class="social-link"><i class="fab fa-instagram"></i></a></li>
+            <li class="list-inline-item"><a href=https://www.linkedin.com/in/methi40/ target="_blank" class="social-link"><i class="fab fa-linkedin"></i></a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>c
```
Comment (bvishal8510) [inline core/templates/developer.html]: remove this extra c

**Comment by Sharma-palak** [comment]
@bvishal8510  sir plzz review

**Pull request merged**